src/vanilla/Model/CompositeTypeCs.cs

```diff
@@ -165,9 +165,9 @@ public string Signature
                         {
                             // for people who really want defaults to properties (be aware of the PATCH operation consequences!)
                             var enumProp = property.ModelType as EnumType;
-                            if(property.UseDefaultInConstructor && enumProp.DefaultValue!=null)
+                            if(property.UseDefaultInConstructor && property.DefaultValue!=null)

```

Comment:
weeeeell and now we can undo the branching here - note that also `property.ModelTypeName` should probably point to `enumProp.ClassName`, but not sure... as little code duplication as possible 😉